src/data-context.js

```diff
@@ -9,6 +9,7 @@ export function DataProvider({children}) {
     faker.seed(123);
 
     const data = [...Array(50)].map((item) => ({
```

Comment:
you can segregate the data coming from faker in a differen file for better readability